src/libm/CMakeLists.txt

```diff
@@ -184,14 +184,14 @@ set_target_properties(${TARGET_LIBSLEEF} PROPERTIES
 )
 
 target_compile_definitions(${TARGET_LIBSLEEF}
-  PRIVATE DORENAME=1 ${COMMON_TARGET_DEFINITIONS}
+  PRIVATE DORENAME=1 PUBLIC ${COMMON_TARGET_DEFINITIONS}
 )
 
 if(COMPILER_SUPPORTS_FLOAT128)
   # TODO: Not supported for LLVM bitcode gen as it has a specific compilation flags
   target_sources(${TARGET_LIBSLEEF} PRIVATE sleefqp.c)
   target_compile_definitions(${TARGET_LIBSLEEF}
-    PRIVATE ENABLEFLOAT128=1 ${COMMON_TARGET_DEFINITIONS})
```

Comment:
COMMON_TARGET_DEFINITIONS was removed from this line because it is already added in the previous target_compile_definitions